src/__stories__/list-story.tsx

```diff
@@ -103,6 +106,19 @@ const Template: StoryComponent<Args & {boxed?: boolean}> = ({
                     ),
                 };
                 break;
+            case 'custom element with text':
+                controlProps = {
+                    right: (
+                        <div>
+                            <Inline space={0}>
+                                <Text3 color={colors.error} medium as="p">
+                                    12,00 €
+                                </Text3>
+                            </Inline>
+                        </div>
```

Comment:
The real use case is that some rows have a navigation, that may show a chevron. While also have a money amount text that also may be in a customized color to show an extra charge.

```suggestion
                        <Inline space={0}>
                            <Text3 color={colors.error} medium as="p">
                                12,00 €
                            </Text3>
                            <IconChevron direction="right" color={colors.neutralMedium} />
                        </Inline>
```

This produces something like the following:
![image](https://user-images.githubusercontent.com/4521712/154517314-afb0cb16-895d-4ea3-bebe-cada175df959.png)

Before, the chevron was _always_ centered, but now we will move it to the top. I'm just trying to be sure about that change.

You can find this construction by searching for `<IconChevron` at webapp, being most of them either:
* Surrounded by a `{display: flex; align-items: center; height: 100%}` style, or
* With a shamy and manually hardcoded padding top, tied to the presence/absecnce ot the row subtitle.

Reply:
yes, chevron has to be always aligned to the center

Reply:
Using `to`, `href` or `onPress` is an implementation detail.
But:
* Because it _navigates_, it has a chevron. and
* Because we need to add a red text at the right, we use the `right` prop, containing both.

And:
* Mística has no control about what's _inside_ `right`, and
* The consumer (filling the `right`) has no control over the _outside_ vertical alignement, then
* I see see way to conciliate both.

That's why I proposed to provide the `centerY` prop as a kind of _hint_ from Mística to the consumer, at https://github.com/Telefonica/mistica-web/pull/407

Reply:
Another idea could be to render the Mïstica's chevron if required (based on the current criteria) even if the `right` prop exists

Reply:
I would like to propose something. Could we manage always the chevron with onPress?
![Screen Shot 2022-02-21 at 23 12 14](https://user-images.githubusercontent.com/6722153/155033136-027d90d2-56ea-4588-b22d-30421fb4e54f.png)

In this way, we avoid Sergio's examples and we will have control of this chevron If in the future we want to modify or whatever